multiline.go

```diff
@@ -86,12 +77,21 @@ func (i *Multiline) Prompt(config *PromptConfig) (interface{}, error) {
 		multiline = append(multiline, string(line))
 	}
 
+	// position the cursor on last line of input
+	cursorPadding := 3
+
+	// ignore the empty line in an empty answer
+	if len(multiline) == 1 && multiline[0] == "" {
+		cursorPadding = 2
+	}
+	cursor.PreviousLine(cursorPadding)
```

Comment:
Let's make this simpler to read at the cost of tiny repetition:
```suggestion
	if len(multiline) == 1 && multiline[0] == "" {
		// ignore the empty line in an empty answer
		cursor.PreviousLine(2)
	} else {
		// position the cursor on last line of input
		cursor.PreviousLine(3)
	}
```
I'm curious about where these constants `2` and `3` come from. From the old code, it looks like the number of lines to clear was calculated by taking the length of `multiline` and adding 2 to that (I guess to account for two blank lines). Now, it either deletes 2 or 3 lines up and not more than that. Is the size of `multiline` taken into account somewhere when redrawing?